### DISCUSSION TITLE
Instance tag settings not working in Arc browser

### DISCUSSION THREAD

**Discussion by lengeradam** [arnoudkooi/SN-Utils#498] [General]
This happens only in Arc browser - https://arc.net/
New instance tag is visible in browser window, I can move it but I can't open the settings window.
If I click on it, nothing happens.

**Comment by arnoudkooi** [comment]
For me it doesn't work at all, do you need to enable something?
Could you try right click context menu "Configure InstanceTag"?

**Comment by lengeradam** [comment]
I just had to reload the tabs the first time to see it.
If I try right click context menu "Configure InstanceTag", nothing happens as well.

**Comment by arnoudkooi** [comment]
I'm afraid Arc doesnt support the sidePanel API. (just like Opera and Safari)
I'll see if I can find a workaround, but may take some time.

**Comment by arnoudkooi** [comment]
I have this as a todo, but small user group, priorities etc. Moving it to a discussion. Will als add a know limitation limitations section to the repo.
